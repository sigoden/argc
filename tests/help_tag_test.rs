#[test]
fn test_no_help_subcommand() {
    let script = r###"
# @cmd
cmd() { :; }
    "###;
    snapshot!(script, &["prog"],);
}

#[test]
fn test_add_help_subcommand() {
    let script = r###"
# @help Print help information
# @cmd
cmd() { :; }
    "###;
    snapshot!(script, &["prog"],);
}
