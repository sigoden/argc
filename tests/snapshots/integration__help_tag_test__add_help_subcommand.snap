---
source: tests/help_tag_test.rs
assertion_line: 17
expression: output
---
RUN
prog

STDOUT


STDERR
USAGE: prog <COMMAND>

OPTIONS:
  -h, --help  Print help information

COMMANDS:
  help  Print help information
  cmd


